cmake/CMakeLists.txt

```diff
@@ -553,11 +553,15 @@ if(BUILD_TOOLS)
   install(TARGETS binary2txt DESTINATION ${CMAKE_INSTALL_BINDIR})
 
   include(CheckGeneratorSupport)
-  if(CMAKE_GENERATOR_SUPPORT_FORTRAN AND CMAKE_Fortran_COMPILER)
+  if(CMAKE_GENERATOR_SUPPORT_FORTRAN)
     enable_language(Fortran)
```

Comment:
The only thing I am not sure about is, if this call will be a fatal error if there is no fortran compiler.

Reply:
Just checked yes it does...

Reply:
so we need to check before enabling:
https://cmake.org/cmake/help/v3.10/module/CheckLanguage.html#module:CheckLanguage

Reply:
Fixed.